src/main/java/org/junitpioneer/jupiter/TempDirectory.java

```diff
@@ -345,6 +346,9 @@ private void tryToDeleteOnExit(Path path) {
 				path.toFile().deleteOnExit();
 			}
 			catch (UnsupportedOperationException ignore) {
+				// this exception shall be ignored by purpose this method is only called in the
+				// stream of the method "createIOExceptionWithAttachedFailures".
+				// That method overrules this "UnsupportedOperationException" anyway.
```

Comment:
```suggestion
				// If the `Path` can't be turned into a `File` (which throws the UOE),
				// it can't be registered to be deleted when the JVM terminates.
				// Because deleting on JVM termination is just a last ditch effort and
				// nicety towards the user, it is entirely optional and shouldn't affect
				// the extension's behavior.
```